packages/dashboard/src/pages/NewWelcome/index.tsx

```diff
@@ -0,0 +1,149 @@
+import { delay } from '@masknet/kit'
+import { DashboardRoutes, EnhanceableSite } from '@masknet/shared-base'
+import { makeStyles } from '@masknet/theme'
+import { Checkbox, FormControlLabel, Typography } from '@mui/material'
+import { memo, useCallback, useState } from 'react'
+import { useNavigate, useSearchParams } from 'react-router-dom'
+import urlcat from 'urlcat'
+import { Services } from '../../API.js'
+import { TermsAgreedContext } from '../../hooks/useTermsAgreed.js'
+import { DashboardTrans, useDashboardI18N } from '../../locales/index.js'
+import { WelcomeFrame } from '../../components/WelcomeFrame/index.js'
+import { Article } from './Article.js'
+import { SecondaryButton } from '../../components/SecondaryButton/index.js'
+import { PrimaryButton } from '../../components/PrimaryButton/index.js'
+
+const useStyles = makeStyles()((theme) => ({
+    title: {
+        fontSize: 36,
+        lineHeight: 1.2,
+        fontWeight: 700,
+    },
+    checkboxRow: {
+        marginTop: theme.spacing(1),
+        marginLeft: 0,
+    },
+    label: {
+        fontSize: 14,
+        color: theme.palette.maskColor.second,
+        fontFamily: 'PingFang SC',
+        marginRight: 0,
+        marginLeft: 0,
+    },
+    link: {
+        color: theme.palette.maskColor.main,
+        textDecoration: 'underline',
+    },
+    checkbox: {
+        padding: 0,
```

Comment:
do not use `padding: 0`, but `margin: -...px`. you can try to focus it with keyboard and see the difference